CMakeLists.txt

```diff
@@ -205,6 +205,15 @@ SET(QT_LIBRARIES
 	Qt${QT_VERSION_MAJOR}::Xml
 )
 
+IF(WANT_QT6)
+    # Core5Compat is needed so that we get QTextCodec which is used by
+    # the Hydrogen importer plugin.
+    # See: https://doc.qt.io/qt-6/qtcore5-index.html
+    FIND_PACKAGE(Qt${QT_VERSION_MAJOR} COMPONENTS Core5Compat REQUIRED)
```

Comment:
Do we really need core 5 compat? Can't we foc the violating function instead?

Reply:
Nvm, beyond the scope of this PR

Reply:
In the long run the Hydrogen importer should be updated to work with Qt6 without compat. Until then this was the easiest way to get things building.